packages/types/src/transaction.ts

```diff
@@ -150,6 +150,12 @@ export interface TransactionMetadata {
 
   /** Metadata for the transaction's spans, keyed by spanId */
   spanMetadata: { [spanId: string]: { [key: string]: unknown } };
+
+  /** Metadata representing information about transaction name changes  */
+  nameChanges: TransactionNameChange[];
+
+  /** The total number of propagations that happened */
+  numPropagations: number;
```

Comment:
This is called `propagations` on `TransactionNameChange`, would opt for using the same shorter name here:

```suggestion
  propagations: number;
```

Reply:
done - https://github.com/getsentry/sentry-javascript/pull/5709/commits/e9ccebda36ffde69f08b1edbddd6ad786db5eca3